nginx/default.conf

```diff
@@ -0,0 +1,12 @@
+server {
```

Comment:
Is it necessary? I reckon that's default configuration of nginx server.

Reply:
you mean the whole file? It might not be 100% needed for now.

What's under location with `try_files $uri $uri/ /index.html;` is definitely necessary to make sure that any route is pointing to the index.html, because it's a SPA (although it has no route other than home yet.

Reply:
Ah, okay. Fair enough!